android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -180,42 +176,41 @@ private void showLocationsOnMap() {
         });
     }
 
-    private void focucOnMarker(Marker marker){
-        marker.showInfoWindow();
```

Comment:
@rohanagarwal94 See https://github.com/Shailesh351/open-event-android/commit/bae48921b798970c01852a7f64c891d2f4e4811e It is working for me.

In my opinion clustering markers is not a good idea because in the event all the location will be nearby.